scripts/deploy/execute-deployment-pipeline.mjs

```diff
@@ -23,16 +23,16 @@ const atomicHostedPage = getVersionComposants(atomicHostedPageJson.version);
 console.log(execSync(`
   deployment-package package create --with-deploy \
     --resolve HEADLESS_MAJOR_VERSION=${headless.major} \
-    --resolve HEADLESS_MINOR_VERSION=${headless.minor} \
-    --resolve HEADLESS_PATCH_VERSION=${headless.patch} \
+    --resolve HEADLESS_MINOR_VERSION=${headless.major}.${headless.minor} \
```

Comment:
Should there be some kind of validation in the script that the `parse` function does not return garbage stuff ?

Reply:
We could add a layer of validation, but IMHO, this would make the script more awkward than it is, for not so much added value.
The parse function worked as intended here, so I don't see that as a need from the incident perpective